pandas/core/frame.py

```diff
@@ -1520,6 +1520,29 @@ def to_feather(self, fname):
         from pandas.io.feather_format import to_feather
         to_feather(self, fname)
 
+    def to_parquet(self, fname, engine=None, compression=None,
+                   **kwargs):
+        """
+        write out the binary parquet for DataFrames
+
+        .. versionadded:: 0.20.0
+
+        Parameters
+        ----------
+        fname : str
+            string file path
+        engine : parquet engine
+            supported are {'pyarrow', 'fastparquet'}
+            if None, will use the option: io.parquet.engine
```

Comment:
Can you use proper punctuation / capital letters here? (in online html docs, line breaks are gone, so this would read strange)